src/main/java/io/xstefank/wildfly/bot/ConfigFileChangeProcessor.java

```diff
@@ -0,0 +1,103 @@
+package io.xstefank.wildfly.bot;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.difflib.UnifiedDiffUtils;
+import com.github.difflib.patch.Patch;
+import com.github.difflib.patch.PatchFailedException;
+import io.quarkiverse.githubapp.ConfigFile;
+import io.quarkiverse.githubapp.event.PullRequest;
+import io.quarkiverse.githubapp.runtime.UtilsProducer;
+import io.quarkiverse.githubapp.runtime.github.GitHubConfigFileProviderImpl;
+import io.quarkus.logging.Log;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.inject.Default;
+import jakarta.inject.Inject;
+import org.kohsuke.github.GHCommitState;
+import org.kohsuke.github.GHEventPayload;
+import org.kohsuke.github.GHPullRequest;
+import org.kohsuke.github.GHPullRequestFileDetail;
+
+import io.xstefank.wildfly.bot.config.RuntimeConstants;
+import io.xstefank.wildfly.bot.config.WildFlyConfigFile;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+@ApplicationScoped
+@Default
```

Comment:
Do you need default if you don't use alternatives?

Reply:
Sorry, trailing from previous testing. I will remove it.